versions.sh

```diff
@@ -62,17 +62,28 @@ for version in "${versions[@]}"; do
 		fullVersion=
 		while [ -z "$fullVersion" ]; do
 			commits="$(
-				wget -qO- "https://cgit.git.savannah.gnu.org/cgit/bash.git/atom/?h=$commit" \
+				{ wget -T2 -qO- "https://cgit.git.savannah.gnu.org/cgit/bash.git/atom/?h=$commit" || \
+					wget -qO- "https://github.com/tianon/mirror-bash/commits/$commit.atom"; } \
 					| "$yq" --input-format xml --output-format json \
```

Comment:
Maybe someday I'll refactor this so it can use the JSON version of GitHub's API, but for now this works with minimal changes. :+1: